draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -49,32 +50,45 @@ const EmptyState: EmptyState = ({
   children,
   straightCorners,
   useZenStyles,
-}) => (
-  <div
-    className={classnames([
-      styles.container,
-      styles[layoutContext],
-      { [styles.straightCorners]: straightCorners },
-    ])}
-    id={id}
-    data-automation-id={automationId}
-  >
-    <div className={styles.illustrationSide}>
-      <img
-        src={illustrations[illustrationType]}
-        className={styles.illustration}
-      />
-    </div>
+}) => {
+  const theme = useTheme()
```

Comment:
It doesn't look like you're using this `theme` variable.

Reply:
Assuming Zen and Heart illustrations are the same dimensions (I think they are?) we can push the responsibility of figuring out what Illustration to render to the Illustration component, and use that component here instead of hosted-assets (though that is a huge improvement on the previously inlined assets).

Reply:
yeah sounds good! keeps everything consistent with other illustrations too, i.e. just dropping in the component.